lib/wasi/Cargo.toml

```
@@ -50,7 +50,7 @@ tracing-wasm = "0.2"
 default = ["sys-default"]
 wasix = []
 
-webc_runner = ["webc", "serde_cbor", "anyhow", "serde", "wasmer/compiler", "wasmer/cranelift"]
+webc_runner = ["webc", "serde_cbor", "anyhow", "serde", "wasmer/compiler", "wasmer/cranelift", "wasmer-vfs/webc-fs"]
```

We should fix this. The runner should never require a compiler or cranelift to run.

```suggestion
webc_runner = ["webc", "serde_cbor", "anyhow", "serde", "wasmer-vfs/webc-fs"]
```